community/modules/compute/schedmd-slurm-gcp-v6-nodeset/variables.tf

```diff
@@ -547,7 +547,7 @@ variable "maintenance_interval" {
 variable "startup_script" {
   description = "Startup script used by VMs in this nodeset"
   type        = string
-  default     = "# no-op"
+  default     = null
```

Comment:
Why change this? Please revert, unless you needed, in this case lets discuss it.

Reply:
This was removed from PR